scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
> net.ifnames=0

IIRC that option was added to systemd/udev in a version later then is included in `jessie`. It used to be mentioned in that wiki, but I guess they _assumed_ that everyone would have that version by now. Maybe webarchive has an old version if you really curious.